### ISSUE TITLE
When Booting to Bootra1n getting ACPI Error thingy

### ISSUE THREAD

**Issue by DEALWITHIT199** [foxlet/bootra1n#97]
When I boot to bootra1n there are some text saying error and saying ACPI Error??? Anyone help me?

**Comment by Checkrane** [comment]
Error -20 Jailbreaking iPad Pro 1st gen On iPadOs 14.0 using Checkra1n checkra1n 0.1 video link of the issue
https://youtu.be/6Oi8LfpUa6Q

**Comment by DEALWITHIT199** [comment]
> Error -20 Jailbreaking iPad Pro 1st gen On iPadOs 14.0 using Checkra1n checkra1n 0.11.0 video link of the issue
> https://youtu.be/6Oi8LfpUa6Q

your comment has nothing to do with my issue.

**Comment by Dark42ed** [comment]
@DEALWITHIT199 Have you tried reading README.md?
![image](https://user-images.githubusercontent.com/74568473/100548607-9d647d80-323b-11eb-82bd-79346157d817.png)

**Comment by DEALWITHIT199** [comment]
> @DEALWITHIT199 Have you tried reading README.md?
> ![image](https://user-images.githubusercontent.com/74568473/100548607-9d647d80-323b-11eb-82bd-79346157d817.png)

Yes I already tried that, it doesn’t work. But you don’t have to worry about it because ACPI Error is normal and its not harmful to the computer.

**Comment by Dark42ed** [comment]
Then why are you asking for help? You are asking like this is a critical error.

**Comment by tyxperr** [comment]
> Then why are you asking for help? You are asking like this is a critical error.

Hey I need some help I tried it and it won’t work do I need to put the code in a specific place?
